src/stores/datStore.ts

```diff
@@ -30,14 +31,46 @@ export interface SellEvent {
     hash: string;
 }
 
+interface DatParams {
+    minInvestment?: BigNumber;
+    currentPrice?: BigNumberCached;
```

Comment:
Since currentPrice isn't a configurable parameter, maybe should rename this to something else . . maybe DatInfo?  I guess price is state, and minInvestment is a param so a little tricky

Reply:
Sure